build/ng-csv.js

```diff
@@ -131,9 +131,16 @@ angular.module('ngCsv.services').
 
           infoArray = [];
 
-          angular.forEach(row, function (field, key) {
-            this.push(that.stringifyField(field, options));
-          }, infoArray);
+          if (options.columnOrder){
```

Comment:
It would be better to have only one forEach here, 
do something like 

```javascript
var iterator = !!options.columnOrder ? options.columnOrder : row;
angular.forEach(iterator, ...
```